### ISSUE TITLE
I am having problems with gulp font-edit

### ISSUE THREAD

**Issue by johnhowson** [kevinchappell/formBuilder#239]
Hi Kevin
Sorry to bother you but I have been trying to add icons to the left bar on Formbuilder but gulp font-edit is not working for me.
I am guessing that in fact what is happening is that the exec command is failing on curl but I am not sure because all I am getting is "Finished 'font-edit' after 8.98 ms"
I have tried adding some debug lines which confirm that I am getting as far as the exec command. I also tried a fresh install just in case it was something to do with my setup. I am running from localhost.
Just really wondered if it is something you have come across before or have any ideas. It also occurred to me that i could manually add the fonts I required. But doing this proved far more difficult than I thought and did not work.
Any ideas very gratefully received
Thanks
John

**Comment by kevinchappell** [comment]
What platform are you developing on? I think Mac is the only one I hadn't tested since I currently don't have one and it sounds like it's stopping at the part of the script that opens the browser. I'll do some debugging on a Mac tomorrow and either fix or let you know what I find.

**Comment by johnhowson** [comment]
Hi Kevin
In fact I am on a Windows machine running Windows 10
I am testing in Apache set up in plain vanilla i.e. at C://Apache24/htdocs
As you say it is stopping at the part that opens the browser and I did wonder if it was because Edge and not Chrome was my default browser so I changed that.
I also wondered if I was somehow missing a module or paths were not correct but I have ruled out the latter I think.

**Comment by johnhowson** [comment]
Hi Kevin 
Thanks for the bugfix unfortunately that did not work either.
Funnily 'start chrome' works when I run it from the $ prompt.
But something is failing on the exec line

**Comment by kevinchappell** [comment]
ok i'll have a closer look at it today

**Comment by johnhowson** [comment]
Thanks a lot much appreciated

**Comment by kevinchappell** [comment]
Finally got around to resolving this issue. fontello should now open in default browser on all platforms.

**Comment by johnhowson** [comment]
Brilliant thanks Kevin
With a bit of messing around it worked.
However font-save quit with an error but I managed to get it to work anyway by unzipping the files manually then running font-save again. 
I noticed the code changed a fair bit and I like the new style of Formbuilder. 
I have been working on a few things and some of this may be of general use to the project (e.g. I needed a validation input for validation messages and also needed some new controls). Haven't got round to putting these into a form that could be generally useful but I hope to do that
